src/app/pages/questions/services/audio-record.service.ts

```diff
@@ -21,90 +17,50 @@ export class AudioRecordService {
   setAudioRecordStatus(status) {
     this.audioRecordStatus = status
   }
+
   getAudioRecordStatus() {
     return this.audioRecordStatus
   }
 
-  startAudioRecording(configFile) {
-    this.recording = this.getAudioRecordStatus()
+  startAudioRecording() {
+    this.audio = new Media(this.getPath() + this.fileName)
+    const options = { SampleRate: 16000, NumberOfChannels: 1 }
+    const recording = this.getAudioRecordStatus()
 
-    if (this.recording === false) {
-      this.recording = true
-      this.setRecordTimer()
-      this.setAudioRecordStatus(this.recording)
-      opensmile.start(this.filename, configFile, this.success, this.failure)
-    } else if (this.recording === true) {
-      this.recording = false
-      this.setAudioRecordStatus(this.recording)
-      opensmile.stop('Stop', this.success, this.failure)
-      this.readAudioFile(this.filename)
+    if (recording === false) {
+      this.setAudioRecordStatus(true)
```

Comment:
is this setAudioRecordStatus() method called somewhere outside this service. Why not just use the variable to set values, instead of using the setter and getter inside the class?